### PR TITLE
codemeta downloadUrl

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "codeRepository": "git+https://github.com/cta-observatory/lstmcpipe.git",
     "dateCreated": "2019-09-09",
     "datePublished": "2021-02-18",
-    "downloadUrl": "https://github.com/cta-observatory/lstmcpipe/archive/refs/tags/v0.5.1.tar.gz",
+    "downloadUrl": "https://github.com/cta-observatory/lstmcpipe/archive/master.tar.gz",
```
Comment (garciagenrique) [inline codemeta.json:8]: Should it be pointing to master or to the latest stable version ?

Comment (vuillaut) [reply codemeta.json:8]: latest version would be nice but since we always forget to update it...

**Review by garciagenrique — CHANGES_REQUESTED** [review]
Please refer to latest release (v0.6.0), and next time we will pay attention to update this field too.

File: codemeta.json (hunk 1 above)
Comment (vuillaut) [reply codemeta.json:8]: Honorable thought, I just don't see it happening, and more importantly I don't want to increase human maintenance cost.

**Comment by vuillaut** [comment]
Will be updated for the next release.

**Pull request closed without merging**